ISSUE 7: Content Edits

daviserinanderson opened the pull request:
Updated content to fix typos & other semantic issues:
- added a break in the first paragraph to highlight code of conduct language
- edited punctuation where needed 
- edited common nouns to lower case
- ensured consistent capitalization of "GitHub"

Quick note. There are some instances in this file where I can't tell if a noun is common or proper. Further edits may be needed.

Commit:
Addressing ISSUE 7: Content Edits

Updated content to fix typos & other semantic issues:
- added a break in the first paragraph to highlight code of conduct language
- edited punctuation where needed
- edited common nouns to lower case
- ensured consistent capitalization of "GitHub"

Quick note. There are some instances in this file where I can't tell if a noun is common or proper. Further edits may be needed.
- updated all mentions of GitHub to show correct capitalization

DiegoPino commented:
@daviserinanderson great! Give me 12-24 hours to read through all your changes. This looks good!

DiegoPino commented on docs/giveortake.md:
```diff
@@ -1,52 +1,55 @@
 ## Where The Wild Things Live
 
-Archipelago has 5 active Github repositories
+Archipelago has 5 active GitHub repositories
```
Totally! So funny i have been using Github instead if GitHub for so long!

DiegoPino commented on docs/giveortake.md:
```diff
@@ -72,10 +75,10 @@ $ git checkout -b ISSUE-6
-- If modifying a `MARKDOWN` file, make sure it renders Ok (you can use Textmate, Atom, Textile, etc to preview) and that links are not broken.
-- If writing large pieces of Code, add PHP Tests. We can help if you don't know how (tell us in the ISSUE). Test will be enforced starting with the first stable release, long conversation of why!
-- If modifying `PHP`, please test your changes live on your local instance of Archipelago. All non Documentation modules are already there inside `web/modules/contrib/`
+- If modifying `PHP`, run `phpcs --standard=Drupal yourchanged.file.php`. We (try our best to) use Drupal 8 \cCoding standards
```
There an extra \c before Coding Standards.

DiegoPino replied:
Probably every item in this list needs to end with a dot.

DiegoPino reviewed (changes requested):
This is great @daviserinanderson. I found only two things that need your attention, please given them a look and once that is done i will merge!

Commit:
Addressed Code Review

Removed the extra \c before coding standards and added periods to all bullets under the section "Do Some Clean Up and Test Locally."

DiegoPino reviewed (approved):
This looks perfect now @daviserinanderson, thanks for addressing my review! 😺
